docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -39,7 +39,7 @@
 version: '3'
 x-airflow-common:
   &airflow-common
-  image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:master-python3.8}
+  image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:|version|}
```

Comment:
I am afraid this is used as "downloadable" piece of code, so we should not replace it here. Maybe we should instead add `stable` tag in DockerHub and refer to it here.

Reply:
Downloadable files are available at airflow.apache.org. Here we have the source code which may contain dynamically generated values. We shouldn't encourage end-users to download this file, so I don't see the problem.

Reply:
Why would the user want to download the file from there instead of airflow.apache.org?

Reply:
I am afraid this is what our documentation encourages user to currently:

https://airflow.apache.org/docs/apache-airflow/stable/docker-compose.yaml

Reply:
(Unless it is also replaced there of course via sphinx)

Reply:
Can you look at changes in docs/conf.py file?

Reply:
Yes, one of the custom plugins replaces `|version|` in the downloadable example files https://github.com/apache/airflow/pull/15484/files#diff-4cd7637805a295ed7c8e92bbd346f5dcbc0e8cbee512f67838f264075904430bR21-R33